apps/benchmarking-image-features/CMakeLists.txt

```diff
@@ -0,0 +1,49 @@
+INCLUDE(../../cmakemodules/AssureCMakeRootFile.cmake)
+INCLUDE(../../cmakemodules/script_qt.cmake)
+
+project(benchmarkingImageFeatures_GUI)
+
+
+
+if(CMAKE_MRPT_HAS_Qt5)
```

Comment:
If your code makes direct calls to opencv, as seems since you link against opencvlibs, then you should also check here for opencv, and skip the project if it's not found

Reply:
Sure I will do this